Update/some_011.txt

```diff
@@ -762,46 +762,46 @@ void main()
 	}
 	ClearMessage();
 
 //r憧れていた頃は、こんなことを考えもしなかった。遠足や林間学校では、自分の好きな子の名前をお互いに言い合って、想いを打ち明けて、それを周りではやし立てる。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "憧れていた頃は、こんなことを考えもしなかった。遠足や林間学校では、自分の好きな子の名前をお互いに言い合って、想いを打ち明けて、それを周りではやし立てる。",
-			NULL, "When I was longing for him, I never thought about things like this. Going on outdoor school trips, asking each other our favorite kids names, sharing our thoughts, joking around.", Line_Normal);
+			NULL, "I'd never thought about things like this when I was yearning for him. On field trips or camps, we'd tell each other the names of our crushes, share our feelings, and tease one another about them.", Line_Normal);
 	ClearMessage();
 
 //rだからその対象が誰であっても、スタートラインはみんな同じ。そして好きな人が誰かと一緒でも、それぞれ頑張りましょうと本心から応援しあうことができた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "だからその対象が誰であっても、スタートラインはみんな同じ。そして好きな人が誰かと一緒でも、それぞれ頑張りましょうと本心から応援しあうことができた。",
-			NULL, "So whatever the target is, the starting line is the same. And I'll be together with my favorite person. We can cheer each other on and offer support from the bottom of our hearts.", Line_Normal);
+			NULL, "That way, we were all at the same starting line no matter who we had our eyes set on. And even if our crushes were with someone else, we could cheer each other on and offer support from the bottom of our hearts.", Line_Normal);
 	ClearMessage();
 
 //rでも、いざ想いが成就しても、お互いに同じ心境でいられるだろうか？…無理だろう。その答えが、今の私のこの気持ちなんだから。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "でも、いざ想いが成就しても、お互いに同じ心境でいられるだろうか？…無理だろう。その答えが、今の私のこの気持ちなんだから。",
-			NULL, "But even if my feelings are fulfilled, can we really be in the same mind as each other? ...That's probably impossible. Primarily because of the way I'm feeling right now.", Line_Normal);
+			NULL, "But now that my feelings have been requited, can we all stay in the same mindset? ...That's probably impossible. Primarily because of the way I'm feeling right now.", Line_Normal);
 	ClearMessage();
 
 //r想いの届かなかった人は成就した相手に嫉妬して、その不幸を望むようになる。そして届いた人も優越感より失うことの焦燥と猜疑に駆られ、周囲の存在が気になりはじめる。
 	if (GetGlobalFlag(GADVMode)) {
 		OutputLineAll("", NULL, Line_ContinueAfterTyping);
 		OutputLine(NULL, "想いの届かなかった人は成就した相手に嫉妬して、その不幸を望むようになる。そして届いた人も優越感より失うことの焦燥と猜疑に駆られ、周囲の存在が気になりはじめる。",
-				NULL, "<size=-2>People who don't receive the feelings of their sweethearts grow to envy their fulfilled partners and wish suffering on them. I'll begin to worry that a person will come for me, driven by scorn and doubt from losing something precious.", Line_Normal);
+				NULL, "<size=-2>Those plagued by unrequited love envy their crush's significant other and wish misfortune upon them. In turn, said significant other, rather than revelling in their success, become paranoid of losing what they have, and start eyeing everyone around them with fear and suspicion.", Line_Normal);
```

Comment:
```suggestion
				NULL, "<size=-2>Those plagued by unrequited love envy their crush's significant other and wish misfortune upon them. In turn, said significant other, rather than revelling in their success, becomes paranoid of losing what they have, and start eyeing everyone around them with fear and suspicion.", Line_Normal);
```